ote/src/cljs/ote/views/own_services.cljs

```diff
@@ -103,23 +131,73 @@
            [delete-service-action e! row]]])
        services))])
 
-(defn transport-services-listing [e! transport-operator-id services section-label]
+(defn- route-error
+  [errors info-open?]
+  (let [error-keys (reduce
+                     (fn [set error]
+                       (conj set (:error error)))
+                     #{}
+                     errors)]
+    [info/info-toggle
+     (tr [:own-services-page :missing-service-info])
+     [:div
+      [:h5 {:style {:margin-top 0}} (tr [:own-services-page :flaws])]
+      (doall
+        (for [error errors]
+          ^{:key (:name error)}
+          [:p {:style {:margin 0}}
+           [:strong [ic/alert-warning {:style {:color colors/negative-button
+                                               :margin-bottom "-5px"}}] (:name error)]
+           " • " (tr [:own-services-page :missing-schedule])]))
+      [:h5 (tr [:own-services-page :guide-to-completion])]
+      [:span
+       [:strong (tr [:own-services-page :add-schedule-to-service]) " "]
+       (tr [:own-services-page :add-schedule-to-service-2]) " ("
+       [ic/content-create {:style {:margin-bottom "-5px"}}]
+       ") " (tr [:own-services-page :add-schedule-to-service-3])
+       [:strong " " (tr [:service-viewer :published-interfaces])]]
+      [:h5 (tr [:own-services-page :schedule-creation])]
+      [:p {:style {:margin-top 0}}
+       (tr [:own-services-page :if-no-interfaces])]
+      (when (error-keys :no-schedule-road)
+        [:p {:style {:margin 0}}
+         [:strong (tr [:enums ::t-service/transport-type :road]) " • "]
+         [linkify "https://www.traficom.fi/fi/asioi-kanssamme/saannollisen-henkiloliikenteen-reitti-ja-aikataulutiedon-digitoiminen"
+          (tr [:own-services-page :open-rae])
+          {:target "_blank"}]])
+      (when (error-keys :no-schedule-sea)
+        [:p {:style {:margin 0}}
+         [:strong (tr [:enums ::t-service/transport-type :sea]) " • "]
+         [linkify "/#/routes" (tr [:own-services-page :open-sea-rae]) {:target "_blank"}]])]
```

Comment:
I think that target _blank is not needed.

Reply:
Is needed to open link in new tab? At least `ote.ui.common.linkify` has `noreferrer` handling for that.